src/fragments/lib/datastore/ios/data-access/save-snippet.mdx

```diff
@@ -5,9 +5,9 @@
 ```swift
 do {
     try await Amplify.DataStore.save(
-        Post(title: "My first post", 
-            content: "Amplify.DataStore is awesome!", 
-            status: .draft)
+        Post(title: "My first post",
+             status: .active,
+             content: "Amplify.DataStore is awesome!")
```

Comment:
also, Xcode complains about hierarchy order for me:

`_Incorrect argument labels in call (have 'title:status:content:', expected 'id:title:content:createdAt:updatedAt:draft:rating:status:comments:')_`

Reply:
we're probably using different schema, is it different from http://localhost:3000/lib/datastore/getting-started/q/platform/ios/#sample-schema ?